content/en/community/hackathons/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Hackathons
+date: 2022-03-31T03:40:00+02:00
+draft: false
+collapsible: true
```

Comment:
Can you add a `description:` attribute with a short sentence summarising this page?